server/dummyControllers/requestController.js

```diff
@@ -0,0 +1,40 @@
+import Request from './../dummyModels/request';
+import User from './../dummyModels/user';
+
+/**
+ * it is a class that control all request api;
+ */
+class requestController {
```

Comment:
Class names should be in PascalCase not camelCase... refer to the Airbnb style guide